### DISCUSSION TITLE
Error: Expecting value: line 1 column 1 (char 0) python program

### DISCUSSION THREAD

**Discussion by ghost** [nianeyna/ao3downloader#90] [Q&A]
Hello,

Today my computer froze during a download and when I was also moving files to Calibre at the same time. I started to get an "Expecting value: line 1 column 1 (char 0) python program" error every time I tried to start a download again. I tried to look it up myself but I couldn't understand any of the solutions that were posted. But this is a JSON Decode Error apparently? Any help would be greatly appreciated.

I could maybe re-download the program but when I was setting it up I had so many problems with no bS4 error etc. and a bunch of other things as well.

Would it be easier to re-download the program?

Thank you!

**Comment by nianeyna** [comment]
If it's indeed a JSON decode error, it's most likely a corrupted log file. Open up the logs folder and try moving log.jsonl to somewhere else (or just renaming it to something like old_log.jsonl)

**Reply by ghost** [reply]
Thank you. I actually ended up re-downloading the program since I couldn't figure it out and it worked fine. If this error happens again I will use your method for it.

Thank you for your time.
